src/dmd/func.d

```diff
@@ -3733,12 +3735,14 @@ extern (C++) class StaticDtorDeclaration : FuncDeclaration
 
     extern (D) this(const ref Loc loc, const ref Loc endloc, StorageClass stc)
     {
-        super(loc, endloc, Identifier.generateIdWithLoc("_staticDtor", loc), STC.static_ | stc, null);
+        this(loc, endloc, "_staticDtor", stc);
```

Comment:
What happens now with multiple module constructors in the same file?

Reply:
Nothing changes, because it forwards the call to the other ctor overload, which still uses `generateIdWithLoc`